packages/syft/src/syft/core/tensor/fixed_precision_tensor.py

```diff
@@ -0,0 +1,86 @@
+# stdlib
+from typing import Any
+from typing import Optional
+
+# third party
+from google.protobuf.reflection import GeneratedProtocolMessageType
+import numpy as np
+
+# syft absolute
+from syft.core.tensor.passthrough import PassthroughTensor
+
+# relative
+from ...core.common.serde.serializable import Serializable
+from ...proto.core.tensor.fixed_precision_tensor_pb2 import (
+    FixedPrecisionTensor as FixedPrecisionTensor_PB,
+)
+from ..common.serde.deserialize import _deserialize as deserialize
+from ..common.serde.serializable import bind_protobuf
+from ..common.serde.serialize import _serialize as serialize
+
+
+@bind_protobuf
+class FixedPrecisionTensor(PassthroughTensor, Serializable):
+    def __init__(
+        self, value: Optional[Any] = None, base: int = 10, precision: int = 3
```

Comment:
Same here , could we have default base as 2?

Reply:
This is used for the moment since it would be simpler to visualize the result (if you have a float number)